packages/stream-model-instance-handler/src/__tests__/model-instance-document-handler.test.ts

```diff
@@ -121,20 +89,50 @@ const jwsForVersion1 = {
   ],
 }
 
-const ThreeIdResolver = {
-  '3': async (did) => ({
+function resolutionResult(didUrl: string, optRotateDate?: string): DIDResolutionResult {
+  const rotateDate = optRotateDate || new Date('2022-03-11T21:28:07.383Z').toISOString()
+  const { did } = parseDidUrl(didUrl)
+  const isVersion0 = /version=0/.exec(didUrl)
+
+  if (isVersion0) {
+    return {
+      didResolutionMetadata: { contentType: 'application/did+json' },
+      didDocument: wrapDocument(
+        {
+          publicKeys: {
+            signing: 'zQ3shwsCgFanBax6UiaLu1oGvM7vhuqoW88VBUiUTCeHbTeTV',
+            encryption: 'z6LSfQabSbJzX8WAm1qdQcHCHTzVv8a2u6F7kmzdodfvUCo9',
+          },
+        },
+        did
+      ),
+      didDocumentMetadata: {
+        nextUpdate: rotateDate,
+      },
+    }
+  }
+
+  return {
     didResolutionMetadata: { contentType: 'application/did+json' },
     didDocument: wrapDocument(
       {
         publicKeys: {
           signing: 'zQ3shwsCgFanBax6UiaLu1oGvM7vhuqoW88VBUiUTCeHbTeTV',
-          encryption: 'z6LSfQabSbJzX8WAm1qdQcHCHTzVv8a2u6F7kmzdodfvUCo9',
+          encryption: 'z6MkjKeH8SgVAYCvTBoyxx7uRJFGM2a9HUeFwfJfd6ctuA3X',
```

Comment:
That's a bit concerning. Why has the encryption key changed?

Reply:
I don't think it did, I think it's just weird with git. This is the encryption key for v1, while `z6LS` is for v0.

Reply:
Here's the line it should be matched with https://github.com/ceramicnetwork/js-ceramic/pull/3062/files/8c235b419ac580a3ec623322972796a3c973dccb#diff-1d5070d6815fcf983adea9a7b864fc0fe112d98ac616adac272bd9d860758f5cL163